### ISSUE TITLE
Figure won't move past a point

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/coulombs-law#86]
For https://github.com/phetsims/QA/issues/221.  Issue for keyboard nav that was encountered on Edge in the XHTML version but was unable to replicate. The figure on the right in Macro Scale was unable to move to the left past a certain point. I had been in full screen before, and the point it stopped was a bit to the right of the the middle of the screen. Dragging by mouse was not effected. After waiting some time after refreshing the page I found the same occurance with the figure on the the left.  Unfortunately, I haven't been able to find a reliable way to  replicate. @jessegreenberg and @mbarlow12 I will add information as it appears.  Made issue as it appeared twice fairly quickly.

**Comment by KatieWoe** [comment]
Now occurred on Win 7 IE, non-XHTML.  May be connected to moving to the atomic scale screen and back.

**Comment by arouinfar** [comment]
@KatieWoe some screenshots of where the figure gets stuck would be helpful. Were the charges close to one another?

**Comment by KatieWoe** [comment]
The placement of the other charge didn't really matter, and the place where the charge gets stuck was different each time.  The first time, q2 was about here.  But I saw q1 get stuck at both ~3.5 and ~6.5 (from memory, not exact).
![stuck locate](https://user-images.githubusercontent.com/41024075/49163471-67a5fa80-f2ea-11e8-862f-e462e454e68d.PNG)

**Comment by jessegreenberg** [comment]
I spent about 5 minutes playing with the charges trying to reproduce with xhtml/coulombs-law_all_phet.html out of master on Edge and could not reproduce. Ill try the RC.

**Comment by KatieWoe** [comment]
Yeah, I've had trouble reliably reproducing it.  Sorry

**Comment by jessegreenberg** [comment]
I was able to reproduce in the RC with this configuration:
![capture](https://user-images.githubusercontent.com/6396244/49167073-a778dc00-f302-11e8-809a-f431ff0cae88.PNG)

I was in full screen mode when the issue first happened, and after exiting full screen mode the problem persisted.

EDTI: This was in Edge.

**Comment by jessegreenberg** [comment]
Same configuration, problem occurred again in master, but the charge stopped at the 5.4 mark.
EDTI: This was in Edge.

**Comment by jessegreenberg** [comment]
I was just able to reproduce this in Chrome in master, and found that the max attribute on the slider isn't set correctly
![capture](https://user-images.githubusercontent.com/6396244/49168650-f70cd700-f305-11e8-973c-eef6ca8288f3.PNG)

This was the setup, the max attribute on the input should be more like 0.013 (left edge of charge 2)
![image](https://user-images.githubusercontent.com/6396244/49168694-0db32e00-f306-11e8-8b65-291c82e848b0.png)

**Comment by jessegreenberg** [comment]
When a charge is zero, the enabled ranges aren't changing. It looks like updateRange is called when force changes and radius changes. 

There is a comment here:
```
    // However, we know the force will change when either of these attributes change, so we can link to that instead of
    // linking to the four underlying attributes. Radius can change without changing force, so it must be linked to
    // radius of each object
```

I guess this actually isn't true when charge is zero, so we need to update the range when position changes too. @mbarlow12 does this seem reasonable?

**Comment by jessegreenberg** [comment]
I am thinking something like this, but haven't been able to test yet because of #92 
```js
    // if positions change while force is zero, we need to update the ranges
    Property.multilink( [ object1.positionProperty, object2.positionProperty ], function ( object1Position, object2Position ) {
      if ( self.forceProperty.get() === 0 ) {
        updateRange( object1 );
        updateRange( object2 );
      }
    } );
```

**Comment by KatieWoe** [comment]
It does actually occur on atomic scale as well

**Comment by mbarlow12** [comment]
@jessegreenberg That sounds reasonable. I think that the function above could entirely replace the foceProperty linked function. Since we're linking to the radius properties anyway, I don't see why we should link to the derived property at all. What do you think?

**Comment by jessegreenberg** [comment]
Sounds good, I agree that position and radius seems sufficient.

**Comment by KatieWoe** [comment]
1.0.0-rc.6 this seems fixed.
